### PR TITLE
cli: fix changelog hyperlink in the log message

### PR DESCRIPTION
**Pull request by mayank**
### Changelog

- [ ] `CHANGELOG.md` is updated with user-facing content relevant to this PR. If no changelog is required, then add the `no-changelog-required` label.

### Affected components

- [ ] Server
- [ ] Console
- [x] CLI
- [ ] Docs
- [ ] Community Content
- [x] Build System
- [ ] Tests
- [ ] Other (list it)

### Related Issues

cli: fix changelog hyperlink in the log message #6682

### Solution and Design

added special character to link so it will stand out in the terminal.

#### Catalog upgrade

Does this PR change Hasura Catalog version?
- [x] No
- [ ] Yes
  - [ ] Updated docs with SQL for downgrading the catalog 

#### Metadata

Does this PR add a new Metadata feature?
- [x] No
- [ ] Yes
  - Does `run_sql` auto manages the new metadata through schema diffing?
    - [ ] Yes
    - [ ] Not required
  - Does `run_sql` auto manages the definitions of metadata on renaming?
    - [ ] Yes
    - [ ] Not required
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?
    - [ ] Yes
    - [ ] Not required

#### GraphQL
- [x] No new GraphQL schema is generated
- [ ] New GraphQL schema is being generated:
   - [ ] New types and typenames are correlated
   
   

#### Breaking changes

- [x] No Breaking changes
- [ ] There are breaking changes:

  1. Metadata API

     Existing `query` types:
     - [ ] Modify `args` payload which is not backward compatible
     - [ ] Behavioural change of the API
     - [ ] Change in response `JSON` schema
     - [ ] Change in error code
     

  2. GraphQL API

     Schema Generation:
     
     
     - [ ] Change in any `NamedType`
     - [ ] Change in table field names
     

     Schema Resolve:-
     
     - [ ] Change in treatment of `null` value for any input fields 
     

  3. Logging

     - [ ] Log `JSON` schema has changed
     - [ ] Log `type` names have changed

### CODE REVIEW

File: cli/commands/update-cli.go [hunk 1]
```diff
@@ -91,7 +91,7 @@ func (o *updateOptions) run(showPrompt bool) (err error) {
 				versionToBeInstalled = latestVersion
 			case hasPreReleaseUpdate:
 				o.EC.Logger.Infof(`a new pre-release version is available:
-- %s (changelog: %s)
+- %s (changelog: \u200B%s\u200B)
```
Comment (scriptonist) [inline cli/commands/update-cli.go:94]:
```
o.EC.Logger.Infof("a new pre-release version is available:\n- %s (changelog: %s)", preReleaseVersion.Original(), getChangeLogLink(preReleaseVersion))
o.EC.Logger.Infof(`to update cli to this version, execute:

  hasura update-cli --version %s

`, preReleaseVersion.Original())
```

File: cli/go.sum [hunk 2]
```diff
@@ -1,3 +1,4 @@
+bazil.org/fuse v0.0.0-20160811212531-371fbbdaa898 h1:SC+c6A1qTFstO9qmB86mPV2IpYme/2ZoEQ0hrP+wo+Q=
```
Comment (scriptonist) [inline cli/go.sum:1]: can you do a `go mod tidy` on your branch?

File: cli-ext/package.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
   "description": "A service to generate Hasura action scaffolds",
   "main": "src/server.js",
   "scripts": {
-    "get-shared-modules": "rm -rf src/shared && cp ../console/src/shared ./src/shared -r",
+    "get-shared-modules": "rm -rf src/shared/* && cp -r ../console/src/shared ./src/shared",
```
Comment (scriptonist) [inline cli-ext/package.json:7]:
```suggestion
    "get-shared-modules": "rm -rf src/shared/ && cp -r ../console/src/shared ./src/shared",
```

**Review by scriptonist — CHANGES_REQUESTED** [review]

File: cli/commands/update-cli.go (hunk 1 above)
Comment (mayank) [reply cli/commands/update-cli.go:94]: Yup, that will work too. Your call.

Comment (mayank) [reply cli/commands/update-cli.go:94]: Issue is with change log URL so how would you suggest to be done. Adding spaces will also work.

**Comment by mayank** [comment]
@scriptonist done with the requested changes. How can I test these on my local machine?

**Comment by mayank** [comment]
@scriptonist can you review this please

**Pull request closed without merging**